README.md

```diff
@@ -1 +1,38 @@
-# Group_E_Simulation_Life_Cycle
\ No newline at end of file
+# Group_E_Simulation_Life_Cycle
+Françoise Blanc
+Roman Cardenas Rodriguez
+Ghida Dandan
+Aidan Fahlman
+Bruno St-Aubin
+
+## Description
+Need to Develop this Section
+
+
+## Software Requirements
+1. Users must be able to provide geospatial data as a source for the model generation workflow:
+	1. All geometric primitives (point, line and polygon)
+	2. Spatial network data  
+	3. In all common geospatial data formats (shapefile, geopackages, csv, geojson, etc.)
+2. Users must be able to provide attribute data in common formats (csv, excel, json) for the model generation process.
+3. Users must be able to provide a workflow file that contains the steps for automated model generation
+4. The system must automatically execute the workflow to convert geospatial data into a Cadmium model configuration file.
```

Comment:
Points 3 and 4 are like the final objective of this work. I would move them to the bottom of the list

Reply:
I have no strong feelings about this, I'm good with moving it to the end.

Reply:
Eventually, we'll have to sort the objective and give them a priority for the first deliverable